controls/sshd_spec.rb

```diff
@@ -276,10 +276,10 @@
 
 control 'sshd-26' do
   impact 1.0
-  title 'Server: Disable PAM'
-  desc 'Avoid challenge-response and password-based authentications.'
+  title 'Server: Disable challenge-response authentication'
+  desc 'Avoid challenge-response authentication.'
   describe sshd_config do
-    its('UsePAM') { should eq('no') }
```

Comment:
@artem-sidorenko and @chris-rock yes we can set PAM to yes and remove the check, because we are checking PasswordAuthentication = no and ChallengeResponseAuthentication = no. But we have already the sshd-29, which checks for ChallengeResponseAuthentication = no. i would recommend to remove this PAM check and renumber the rest.